ci/scripts/README.md

```
@@ -0,0 +1,184 @@
+### Test Complete Pipelines
+
+```bash
+# Test transport pipeline
+cd scripts
+node run-pipeline.mjs transport-build core v1.2.3
+node run-pipeline.mjs transport-build transport transports/v1.2.3
+```
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)